helmfile/charts/ipv4/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "ipv4.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
```

Comment:
Ah here is the explanation from my question above I guess.

Reply:
ya, so these are built automatically during the bootstrapping of a helm project (helm create).